frontend/src/Navigation.ts

```diff
@@ -85,3 +94,17 @@ export default async ({ dynamicNav, currentNamespace }: DynamicNav) => {
       }
     : routes;
 };
+
+export const useNavigation = ({ dynamicNav, currentNamespace }: DynamicNav): Navigation | RouteProps[] => {
+  const [navigation, setNavigation] = React.useState<Navigation | RouteProps[]>();
```

Comment:
Undefined state is ignored by the calling code? Haven't we always passed an empty object 🤔 

I guess we are dealing with a different flow, "sync `undefined`" (not) vs "async empty" (before)

Reply:
Yup, exactly. Chrome will start loading when `undefined` is passed, instead of hiding the nav on empty array.